actionlist/app/components/action_item.js

```diff
@@ -76,6 +76,16 @@ const mapDispatchToProps = dispatch => ({
 export default connect(null, mapDispatchToProps)(ActionItem);
 
 ActionItem.propTypes = {
-  rowData: React.PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+  rowData: React.PropTypes.shape({
+    rowData: {
+      expanded: {},
+      title: {},
```

Comment:
Isn't the type of `expanded` a  `React.PropTypes.bool` and `title` a `React.PropTypes.string`?